lib/fishjam/room.ex

```diff
@@ -139,6 +141,12 @@ defmodule Fishjam.Room do
     GenServer.cast(registry_id(room_id), {:media_event, peer_id, event})
   end
 
+  @spec disconnect_peer(socket_pid :: pid(), room_pid :: pid(), reason :: atom()) :: :ok
+  def disconnect_peer(socket_pid, room_pid, reason) do
+    send(room_pid, {:DOWN, :any_ref, socket_pid, reason})
```

Comment:
1. Should be `{:DOWN, ref, :process, pid, reason}`...
2. ...but impersonating a `:DOWN` tuple feels... wrong